app/assets/javascripts/read/src/components/collection-modal.js

```diff
@@ -121,6 +115,7 @@ export default class extends Component {
               pinyin={c.notice.title_p}
             />
 
+            Grave Collection #{this.props.feature.id}
```

Comment:
We do. According to Tom's sketch, it gets moved from the title to the body of the modal.